.github/maintainers-little-helper.yaml

```diff
@@ -50,7 +50,7 @@ flake-tracker:
     - ci/flake
   jenkins-config:
     jenkins-url: https://jenkins.cilium.io
-    regex-trigger: (^test-me-please)
+    regex-trigger: (^/?test(-me-please)?)
```

Comment:
@aanm Will that work as intended?

Reply:
@nbusseneau yes but the right regex would be `(^(/test|test-me-please))` otherwise `/test-me-please` will also be considered.

Reply:
Yes, I thought it was nice having it in to during the transition phase. I also use that same regex allowing both on Jenkins.